cmd/maya-apiserver/cstor-operator/cspc/handler.go

```diff
@@ -160,3 +170,19 @@ func (pc *PoolConfig) create(pendingPoolCount int, cspc *apis.CStorPoolCluster)
 	}
 	return nil
 }
+
+func (pc *PoolConfig) createDeployForCSPList(cspList []apis.NewTestCStorPool) {
+	for _, cspObj := range cspList {
+		cspObj := cspObj
```

Comment:
can this piece of code reusable?

Reply:
If it reusable -- do we want it to put into a separate package?

Reply:
done